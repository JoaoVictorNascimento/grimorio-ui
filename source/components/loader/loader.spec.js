import '../../../internals/test/helper';

import Loader from './loader-component';

/** @test {Loader} */
describe('Loader component', () => {
  /** @test {Loader#render} */
  describe('#render', () => {
    it('render correctly', () => {
      const wrapper = shallow(
        <Loader
          type='full'
          color='secondary'
          background='black'
          size='18'
        />
      );
      expect(wrapper.html()).toMatchSnapshot();
    });
  });
});
