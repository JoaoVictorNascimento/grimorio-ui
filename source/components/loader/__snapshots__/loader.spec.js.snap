// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component #render render correctly 1`] = `"<div class=\\"wrap isFull isBlack\\"><span style=\\"width:18;height:18\\" class=\\"spinner secondary\\"></span></div>"`;
